matsim/src/main/java/org/matsim/core/mobsim/jdeqsim/Road.java

```diff
@@ -352,14 +333,6 @@ public void removeFromInterestedInEnteringRoad() {
 		assert (this.interestedInEnteringRoad.size()==this.deadlockPreventionMessages.size());
 	}
 
-	public static Road getRoad(Id<Link> linkId) {
-		return getAllRoads().get(linkId);
-	}
-
-	public void setLink(Link link) {
```

Comment:
Was this never used? It seems dangerous to remove a public method.

Reply:
Yes, this was never used. And as I understood Road may have only a single link and is not supposed to change the link during a lifetime. I removed this method. Though you are right this may be used for some reason in some other branches/forks. This method is easy to restore.